libs/designsystem/src/lib/components/form-field/affix/input-affix.component.scss

```diff
@@ -0,0 +1,18 @@
+@import '~@kirbydesign/core/src/scss/base/variables';
+
+:host {
+  --input-padding: #{$form-field-input-padding};
+}
+
+.affix {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+
+  &.default {
+    font-size: 16px;
+    line-height: 24px;
+    letter-spacing: 0;
+    color: #707070;
+  }
```

Comment:
Should use design tokens and utility functions - not hardcoded values. E.g. `font-size: utils.font-size('n');`. The same goes for colors.